src/nat/libp2p_nat.erl

```diff
@@ -180,7 +187,9 @@ retry_matrix(Try, Port) ->
 %% @end
 %%--------------------------------------------------------------------
 increment_port(Port) when Port < 65535-1 ->
-    Port+1.
+    Port+1;
+increment_port(_Port) ->
+    65535.
```

Comment:
Just return Port here? Less hardcoding of value?

Reply:
indeed